addons/portal/static/src/xml/portal_security.xml

```diff
@@ -21,6 +21,18 @@
                 and complete, <strong>it will be the only way to
                 identify the key once created</strong>.
             </p>
+            <h3 class="fw-bold">
+                Give a duration for the key's validity
+            </h3>
+            <select name="duration">
```

Comment:
```suggestion
            <select name="duration" class="form-control">
```
To have the nicer bootstrap style
![image](https://github.com/user-attachments/assets/e2a997ff-939a-441f-bbe8-41f95de9f07f)

![image](https://github.com/user-attachments/assets/4f6bf3b8-1a49-48d9-88ff-fdb2cbc7c4c2)